webpack/assets/javascripts/foreman_tools.js

```diff
@@ -103,43 +103,28 @@ export function initTypeAheadSelect(input) {
 // handle table updates via turoblinks
 export function updateTable(element) {
   const uri = new URI(window.location.href);
-  let searchTerm;
-  let perPage;
-  let isSearchForm;
-  let pageNum;
-
-  if (element !== undefined) {
-    isSearchForm = element.id === 'search-form';
-
-    pageNum = $('#cur_page_num').val();
-    if (pageNum !== undefined) {
-      uri.setSearch('page', pageNum);
-    }
-
-    if (isSearchForm || element.id === 'per_page') {
-      uri.setSearch('page', 1);
-    }
-
-    if (isSearchForm) {
-      searchTerm = $(element)
-        .find('.autocomplete-input')
-        .val();
-      if (searchTerm) {
-        uri.setSearch('search', searchTerm.trim());
-      }
-    }
-
-    perPage = $('#per_page').val();
-    if (
-      perPage !== undefined &&
-      $('#search-form')
-        .find('.autocomplete-input')
-        .val() === undefined
-    ) {
-      uri.removeSearch('search');
-    }
-    uri.setSearch('per_page', perPage);
+
+  // if element is undefined don't refresh page or update Turbolinks.
+  if (element === undefined) {
+    return false;
```

Comment:
@ohadlevy can you review this part of the code.
Is it OK that we are not returning `true` and not calling `Turbolinks`?
Please notice the unit-test for `updateTable` when `element === undefined` was changed accordingly.
The reason that we changed it is because we didn't find a reason to do so when `element === undefined` because the `uri` will stay the same (unless was modified by the user manually, i guess).

Reply:
@ripcurld when would it be undefined? based on how its called (in pagination / search) I am not sure it can be undefined?

Reply:
@ohadlevy :+1: 
Exactly, looking at the code `updateTable` is always called with element that is not `undefined`.
But then, looking at the unit tests I saw there was a test where `updateTable` is called without passing any arguments which verifies `Turbolinks` will be called.

Checking in `git` it said you wrote it and I thought maybe I'm missing something and you could determine if this change is fine. :grin: